testing/rabbitmq/APKBUILD

```diff
@@ -0,0 +1,91 @@
+# Contributor: Nathan Johnson <nathan@nathanjohnson.info>
+# Maintainer: Nathan Johnson <nathan@nathanjohnson.info>
+pkgname=rabbitmq
+_realname=rabbitmq-server
```

Comment:
W don't have any rule for using or not using `-server` suffix (as I know), but we have a rule to stick with the upstream's name where possible.